Source/Scene/GlobeSurfaceTileProvider.js

```diff
@@ -130,9 +142,16 @@ define([
         this.showGroundAtmosphere = false;
         this.shadows = ShadowMode.RECEIVE_ONLY;
 
-        this.hueShift = undefined;
-        this.saturationShift = undefined;
-        this.brightnessShift = undefined;
+        /**
+         * The color to use to highlight fill tiles. If undefined, fill tiles are not
+         * highlighted at all. The alpha value is used to alpha blend with the tile's
+         * actual color.
+         */
+        this.fillHighlightColor = undefined;
```

Comment:
This is just for debugging right? Should this be named something like `debugFillHightlightColor` to be consistent with similar things like [Cesium3DTileset.debugColorizeTiles](https://github.com/AnalyticalGraphicsInc/cesium/blob/master/Source/Scene/Cesium3DTileset.js#L669) ?

Reply:
No, I don't see this as only for debugging. I actually considered having it on by default, and I think it might be a good idea to enable it for certain kinds of applications. Fill tiles don't represent the real terrain surface at all, they just fill the space. So it can be useful to visually indicate that a part of the surface is complete nonsense. For example, you might want to darken fill tiles a bit, or maybe even make them appear 100% black. The latter is almost like not rendering the files at all, except that you can't see the stars and atmosphere artifacts through black fill tiles. The effect is similar to what you get in 2D maps when you pan quickly and adjacent tiles aren't loaded yet.